hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -1010,6 +1014,23 @@ public Data executeOnKeyInternal(Data key, EntryProcessor entryProcessor) {
         }
     }
 
+    private static void validateEntryProcessorForSingleKeyProcessing(EntryProcessor entryProcessor) {
+        if (entryProcessor instanceof Offloadable) {
```

Comment:
Fixed here too. BTW, couldn't find a good common place for this method. Now it's duplicated in the client and server proxy.